Update VIP picture and IP address requirements (#468)
harvester/docs: docs/install/requirements.md

Harvester nodes have the following network requirements for installation.

### IP Address Requirements for Harvester Nodes

> 
> Harvester is built on top of Kubernetes, each Node needs an independent IP address, this IP is used for the Node identity, it cannot change during the lifecycle of a Harvester cluster.
> 
> ### IP Address Requirements for Harvester Cluster
> 
> The Harvester cluster needs an additional IP address, it is used for the management IP of the whole cluster, which is also called Virtual IP (VIP).
> 
> For more details, please refer [management address](./management-address.md)
> 

Reviewer: ```suggestion

Harvester is built on top of Kubernetes, and each node needs an independent IP address. Harvester uses this IP address to identify a node identity, and it cannot change during the lifecycle of a Harvester cluster.

### IP Address Requirements for Harvester Cluster

The Harvester cluster needs an additional IP address called Virtual IP (VIP). It uses it as the management IP for the whole cluster.

Please refer to [Management Address](./management-address.md) for more details. 

```
